utils/fs/fs.go

```diff
@@ -108,6 +108,14 @@ func OverlayMount(mountPoint string, lowerDirs []string, workDir, upperDir strin
 func Umount(mountPoint string) (err error) {
 	log.WithFields(log.Fields{"mountPoint": mountPoint}).Debug("Umount dir")
 
+	if _, err = os.Stat(mountPoint); err != nil {
```

Comment:
Logically it is not correct. I guess it should be checked in SM launcher on stop instance before calling umount.